ecosystem/sep-0024.md

```diff
@@ -190,6 +191,24 @@ The deposit flow can only be fulfilled if the Stellar `Account` has established
 5. `Anchor` detects the trust line creation in the `Account`. If the asset is `AUTH_REQUIRED`, `Anchor` approves the new trust line.
 6. `Anchor` proceeds with the deposit flow.
 
+##### Claimable Balances
+
+[Protocol
+14/15](https://www.stellar.org/blog/protocol-14-improvements?locale=en) now allows anchors to
+complete payments using the [ClaimableBalance
+functionality](../core/cap-0023.md) and the user can collect the funds at any
+time.
+
+This is an optional, yet recommended, feature that will no longer need to wait
+for the `Wallet` or the user to establish a
+trust line for the `Account` for the given asset.
+
+1. If the given Stellar `account` does not exist, on receipt of the deposit, the
```

Comment:
I don't think these 2 points are necessary. The logic is simple, no trustline --> claimable balance, trustline --> payment. I wouldn't introduce the "if the stellar account doesn't exist" case.